docs/get-started/quick-start.md

```diff
@@ -23,10 +23,13 @@ Wait until NFD master and worker are running.
 
 ```bash
 $ kubectl -n node-feature-discovery get ds,deploy
-NAME                        DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
-daemonset.apps/nfd-worker   3         3         3       3            3           <none>          5s
+NAME                                  DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
+daemonset.apps/nfd-topology-updater   2         2         2       2            2           <none>          5s
```

Comment:
`default` does not deploy topology-updater

You could add an optional step of deploying topologyupdater overlay